### ISSUE TITLE
Errors: SSL Certificate or Json decoding failed

### ISSUE THREAD

**Issue by unnic** [ibamacsr/sentinelsat#35]
Hello,

I am trying to implement your tool on a new computer. 
running on `scihub.coperinicus/s2/` i get `SSL certificate problem: self signed certificate in certificate chain`

trying with `scihub.coperinicus/dhus/`: API response not valid. JSON decoding failed.

I tried logged in and with the guest//guest account. The JSON is fine and works on another computer with an older version of the sentinelsat.

Any ideas how I can get the new version to run?

**Comment by Fernerkundung** [comment]
This error likely depends on your installation of pyCurl and the CA bundle it uses. Most of the time this issue can be solved by installing `certifi` - as noted in the [troubleshooting section of the docs](http://sentinelsat.readthedocs.io/en/latest/install.html#troubleshooting).

**Comment by unnic** [comment]
I already checked certifi, it's installed and updated.

**Comment by Fernerkundung** [comment]
Did you try passing through a cacert.pem yourself? We had this issue before (#1) and it is not related to `sentinelsat` but rather homura>pyCurl>CA certificates. The solutions mostly circle around using a different version of pyCurl, providing the cacert.pem yourself and/or installing openssl dependencies. All very dependent on your OS, your Python installation, environment, etc.

On Windows I saw best results by using [conda](http://conda.pydata.org/docs/install/quick.html).

I also just verified that this is not a `sentinelsat` issue. The download is working fine on Ubuntu 14.04 and Win7 x64 with Python 2.7 and 3.4.
